langchain4j/src/test/java/dev/langchain4j/service/AiServicesIT.java

```diff
@@ -64,6 +64,26 @@ void afterEach() {
         verifyNoMoreInteractions(moderationModel);
     }
 
+    interface EggCounter {
+
+        @UserMessage("Count number of 'egg' mentions in this sentence:\n|||{{it}}|||")
+        int count(String sentence);
+    }
+
+    @Test
+    void test_simple_instruction_with_primitive_return_type() {
+        EggCounter eggCounter = AiServices.create(EggCounter.class, chatLanguageModel);
+
+        String sentence = "I have ten eggs in my basket and three in my pocket.";
+
+        int count = eggCounter.count(sentence);
+        System.out.println(count);
```

Comment:
minor: please add an assertion on `count` to make sure LLM understands the prompt